src/main/java/mops/gruppen2/domain/model/group/wrapper/Limit.java

```diff
@@ -0,0 +1,27 @@
+package mops.gruppen2.domain.model.group.wrapper;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+@Value
+public class Limit {
+
+    @NotNull
+    @Min(1)
+    @Max(999_999)
+    @JsonProperty("value")
+    long value;
+
+    public static Limit DEFAULT() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name 'DEFAULT' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/tobi_2/gruppenfindung/pullRequest?prid=5356872)